src/workflows/airqo_etl_utils/airqo_api.py

```diff
@@ -372,7 +378,119 @@ def get_cohorts(self, tenant: Tenant = Tenant.ALL) -> list:
             for cohort in response.get("cohorts", [])
         ]
 
-    def get_sites(self, tenant: Tenant = Tenant.ALL) -> list:
+    def get_site(self, site_id: str) -> Dict[str, Any]:
```

Comment:
Hi @NicholasTurner23 , thanks for this. 
Speaking from a high level perspective, why are we getting site details again? 

Could you jog my memory a little about the following?
- The problem statement related to this solution? and
- All the approaches we have considered before arriving at this specific one of getting sites?

Thanks again! I love seeing new PRs. :)

Reply:
Referencing Jira card(TechOps-232).

- The problem statement related to this solution?
a. We had some devices missing latitude and longitude data. 

- All the approaches we have considered before arriving at this specific one of getting sites?
a. Filter out such devices and verify/update their data using site data. And hence the need to be able to get a single site vs getting all sites. This could have been done within the existing `extract_devices_data` or in a different task all together.
b.  The method it's self could be in any other scenarios where one needs to verify a single device's data and or a single site's data

Thank you.

Reply:
Thanks @NicholasTurner23 , the devices which were missing GPS data, 

1. Were they in STAGING or PRODUCTION environment?
2. Were they deployed devices?
3. Can you share a sample of those devices? Their` device_numbers` or `IDs` or `unique names`?

Thanks